cmd/bucket-handlers.go

```diff
@@ -33,7 +32,7 @@ import (
 
 // http://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html
 // Enforces bucket policies for a bucket for a given tatusaction.
-func enforceBucketPolicy(bucket string, action string, reqURL *url.URL) (s3Error APIErrorCode) {
+func enforceBucketPolicy(bucket string, action string, req http.Request) (s3Error APIErrorCode) {
```

Comment:
note: we don't follow this practice in our code base, just a personal observation that it's a better practice